src/core/i18y/index.ts

```diff
@@ -0,0 +1,18 @@
+import I18yStrings from "./I18yStrings";
+
+// Module imports can be done dynamically/lazy in future tasks
+import NL from "./locales/NL";
+import EN from "./locales/EN";
+
+type AvailableLanguages = "EN" | "NL";
+
+const I18Y = (language: AvailableLanguages): I18yStrings => {
+  switch (language) {
+    case "NL":
```

Comment:
In the comment response you mentioned "useState" - I think it would be great to actually have a app-level toggle for languages  switching state :)